package.json

```diff
@@ -10,7 +10,7 @@
     "powerbi-visuals"
   ],
   "scripts": {
-    "postinstall": "pbiviz update 2.1.0",
+    "postinstall": "pbiviz update 2.3.0",
```

Comment:
We can remove post install, no any reason to run update for installing installed powerbi-visuals-api package